Makefile

```diff
@@ -1,6 +1,6 @@
 all: lkmpg.tex
 	pdflatex -shell-escap lkmpg.tex
-
+	make4ht -suf html5 -c config.cfg -d html lkmpg.tex
```

Comment:
`make4ht` should be excluded from default target. Instead, create a new build target `html`.

Reply:
Got it.